src/index.js

```diff
@@ -82,7 +82,8 @@ const QRCode = ({
   linearGradient = ['rgb(255,0,0)', 'rgb(0,255,255)'],
   ecl = 'M',
   getRef,
-  onError
+  onError,
+  testID
```

Comment:
I see you added `testID`, but you didn't pass it to the `Svg` component.
So basically this code won't work